specTemplate.spec

```diff
@@ -33,6 +35,9 @@ BuildRoot: %{_tmppath}/%{_packagename}-%{_version}-%{_release}-buildroot
 Prefix: %{_prefix}
 Requires: __requires_list__
 BuildRequires: __build_requires_list__
+%if %{_is_arm}
+AutoReqProv: no
```

Comment:
`AutoReq` instead of `AutoReqProv`?

Reply:
nope. This is the correct syntax

Reply:
I know it is "a" correct syntax, my question was whether we need to disable the "provides" functionality, or *just* the "requires" functionality

Reply:
hmmm... for some reason the `AutoReq: no` was giving an error before. But not anymore...